Board.js

```diff
@@ -74,6 +88,24 @@ class Board {
 		this.checkWin();
 	}
 
-	checkWin() {
+	checkWin(executive) {
+		if ((this.shipSpaces == 0)){
+			alert("You win!") //Improve: Say which player won
+			executive.renderBothBoards(); //FIX: doesnt render both
```

Comment:
Instead of passing executive and calling a method within it hear, checkWin should return a boolean. Executive can then decide how to display that information with the added benefit of knowing which board it came from. The only reason I passed executive into render was because it seemed to be the only way to attach it to the click listener, but here it's not necessary.